INSTALL.md

```diff
@@ -41,3 +38,28 @@ compile an example
           see [TuningGuide.md](doc/TuningGuide.md)
 - `make -j`
 - `./matrixMul -wA=320 -wB=320 -hA=320 -hB=320` (parameters must be a multiple of 32!)
+
+
+How to update Alpaka as git subtree?
+------------------------------------
+
+```zsh
+## add subtree (not needed, already cloned with git)
+# git subtree add --prefix alpaka https://github.com/ComputationalRadiationPhysics/alpaka.git develop --squash
+## Update
+git subtree pull --prefix alpaka https://github.com/ComputationalRadiationPhysics/alpaka.git develop --squash
```

Comment:
Please update with a generic tool user name and e-mail, see https://github.com/ComputationalRadiationPhysics/picongpu/pull/2893